ONSdigital/census-rm-docker-dev
# Add uac-qid service and remove iac service

# Motivation and Context

Added new uac-qid service and removed IAC service.

# What has changed



* Added uac-qid service config
* Bumped max_connections for postgres as we were seeing intermittent failures when running tests.

# How to test?



* Run `mvn clean install` to create a local Docker image on the below branches:
    * uac-qid-service: `first-version`
    * case-processor: `use-new-uac-qid-service`
* Run `make up` and ensure the services start correctly. 
* Run the acceptance tests and make sure it passes.

# Links



https://trello.com/c/cXw970Gx/902-build-uac-not-iac-qid-service-8
